2025-05-21 00:46:47,974 INFO 56417 [egg-logrotator] agent logger reload: got log-reload message
2025-05-21 01:12:06,353 ERROR 56417 nodejs.ClusterClientNoResponseError: client no response in 1498166ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/zuohuaihan/Desktop/归档/z-blog-server/node_modules/.pnpm/cluster-client@3.7.0/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 56417
hostname: zuohuaihandeMacBook-Pro.local

2025-05-21 01:48:17,517 WARN 56417 [agent:development] reload worker because /Users/zuohuaihan/Desktop/归档/z-blog-server/app/router.js change
2025-05-21 01:48:17,314 INFO 56417 [egg-watcher] Received a change event from eventSource: {"event":"change","path":"/Users/zuohuaihan/Desktop/归档/z-blog-server/app/router.js","stat":{"dev":16777223,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":15611846,"size":512,"blocks":8,"atimeMs":1747763297280.315,"mtimeMs":1747763297247.4043,"ctimeMs":1747763297247.4043,"birthtimeMs":1747045079382.2686,"atime":"2025-05-20T17:48:17.280Z","mtime":"2025-05-20T17:48:17.247Z","ctime":"2025-05-20T17:48:17.247Z","birthtime":"2025-05-12T10:17:59.382Z"},"remove":false,"isDirectory":false,"isFile":true}
